jenkins/build-prod.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+    - name: mvn
+      image: maven:3.6.3-jdk-8
+      resources:
+        requests:
+          cpu: 0.5
```

Comment:
This parameter defines how many pods you can run per VM with 2 cpus, meaning you can run 3 pods(jobs) for one VM. that what you want?

Reply:
We can tweak it later if it's too either too much or not enough

Reply:
ok